src/color.cc

```diff
@@ -20,11 +20,9 @@
 
 #include "wabt/common.h"
 
-#if _WIN32
```

Comment:
CI seems to be passing..  

If this doesn't work we can just drop this PR and leave things as is

Reply:
Oh, huh. I think I just learnt that `#if SOME_UNDEFINED_MACRO` correctly evaluates to false in C. Cool, will land this then.